src/js/game/LevelMVC/FacingDirection.js

```diff
@@ -33,6 +33,15 @@ const FacingDirection = Object.freeze({
       case FacingDirection.East: return [1, 0];
       case FacingDirection.West: return [-1, 0];
     }
+  },
+
+  directionToRelative(direction) {
+    switch (direction) {
+      case FacingDirection.North: return "Up";
+      case FacingDirection.South: return "Down";
+      case FacingDirection.East: return "Right";
+      case FacingDirection.West: return "Left";
```

Comment:
Should these be "North", "South", "East", "West" for consistency?  (Updating the piston block names to match.)

Reply:
I can see the value in updating pistons to use cardinal rather than relative directions, but it's _definitely_ out of scope for this PR

Reply:
Ah right, because it would require changes to the levels as well 😕.  Sounds good!